packages/cli/src/commands/deploy/aws-providers/serverless.js

```diff
@@ -7,25 +7,10 @@ export const preRequisites = [
       'Please follow the steps at https://www.serverless.com/framework/docs/providers/aws/guide/installation/ to install Serverless.',
     ],
   },
-  {
-    title: 'Checking if @netlify/zip-it-and-ship-it is installed...',
-    command: ['yarn', ['zip-it-and-ship-it', '--version']],
```

Comment:
The new deps that have been added are `@vercel/nft`, `archiver` and `fs-extra`. They are not cli tools so can't use `--version` as a test, is there an easy way to check deps?

Reply:
Not sure it is easy, but in the past to verify dependencies like this are there, I have loaded `package.json` in code and inspected the JSON object to verify that there is the expected key/value under `dependencies`/`devDependencies`